lib/bn.js

```diff
@@ -1021,6 +1237,12 @@ BN.prototype.mul = function mul(num) {
   return this.mulTo(num, out);
 };
 
+// Multiply employing FFT
+BN.prototype.mulf = function mulf(num) {
+  var out = new BN(null);
+  return jumboMulTo(this, num, out);
```

Comment:
`N` will be calculated from `this` and `num` in the `jumboMulTo`. You may want to allocate `out.words = new Array((this.length + num.length) * 2)` here.